jobmon_client/src/jobmon/client/node.py

```diff
@@ -101,7 +101,9 @@ def _hash_node_args(self) -> int:
         str_arg_ids = [str(arg) for arg in arg_ids]
 
         hash_value = int(
-            hashlib.sha256("".join(str_arg_ids + arg_values).encode("utf-8")).hexdigest(),
+            hashlib.sha256(
+                "".join(str_arg_ids + arg_values).encode("utf-8")
+            ).hexdigest(),
```

Comment:
I hope black doesn't want to change it back.

Reply:
It was black that originally complained about it lol. This PR is currently passing linting, so fingers crossed!